masonite/request.py

```diff
@@ -194,8 +195,13 @@ def cookie(self, key, value, encrypt=True, path='/', expires=''):
         if expires:
             expires = "Expires={0};".format(cookie_expire_time(expires))
 
+        http_only_text = ""
+        if http_only:
+            http_only_text = "HttpOnly"
+
```

Comment:
hmm I'm not really a big fan of the random `http_only_text` ...

can we do something like:

```python
def cookie(..., http_only='HttpOnly')

if not http_only:
    http_only = ''
...
```

and then the rest looks good.

The idea here is that we developers will never need to specify it as True because it's always true. for example they will never do:

```python
request.cookie('key', 'value', http_only=True)
```

since it's always true by default. So with that we can just specify the string and only check for a falsy value (`False`)

since we set it to an empty string it obviously won't show up when setting the cookie